src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
Woopsie, corrected the links.
Exactly, 20ms should be easily possible.
Unauthenticated requests currently work, but I cannot find any Google documentation confirming this.
I've run into an issue this morning where unauthenticated access worked for some time, but then I started running into "Over query limits". It seems as if I've used up the daily unauthenticated quota, as this was unrelated to the number of queries per second. Retrying in this case would result in the program retrying endlessly. We would therefore also have to implement a "maximum number of retries" limitation.